scripts/migrate-vfolder-structure.py

```diff
@@ -0,0 +1,35 @@
+import sys
+from pathlib import Path
+
+
+def main():
+    input_path = Path(sys.argv[1])  # full path for vroot/local
+    output_path = Path(sys.argv[2])  # full path for volume directory. ex. vfs/
+    print(input_path, output_path)
+    subfolders = [x for x in input_path.iterdir() if x.is_dir()]
+
+    print("Number of folders ", len(subfolders))
+    try:
+
+        for folder in subfolders:
+            folder = str(folder)
+            folder = folder.split("/")[-1]
+            header_dir = Path(folder[0:2])
+            second_lvl_dir = Path(folder[2:4])
+            rest_dir = Path(folder[4:])
+
+            Path(output_path / header_dir).mkdir(parents=True, exist_ok=True)
+            Path(output_path / header_dir / second_lvl_dir).mkdir(parents=True, exist_ok=True)
+            Path(output_path / header_dir / second_lvl_dir / rest_dir).mkdir(parents=True, exist_ok=True)
```

Comment:
And also... path fragments should be treated as strings, and we can exploit the fact that `output_path` is already a Path object:
```suggestion
            header_dir =folder[0:2]
            second_lvl_dir =folder[2:4]
            rest_dir =folder[4:]

            (output_path / header_dir / second_lvl_dir / rest_dir).mkdir(parents=True, exist_ok=True)
```